### DISCUSSION TITLE
I am a beginner dumb and i want to use fabric and i really dont understand how do i download it

### DISCUSSION THREAD

**Discussion by Khuz411** [danielmiessler/Fabric#639] [Q&A]
i really dont understand how do i download it 
i have downloaded wsl used the pipx but when i do fabric --setup nothing happens and i cant do anything i am just too dumb i tried the chatgpt for answer but it cant fix please someone please help me

**Comment by snafu4** [comment]
Certainly not dumb (since you are trying), just inexperienced…

Let’s do this step by step:

Show EXACTLY what you have done (in the order you did it) and the results (are you on Windows 10 or 11?).  You should be following the instructions on the main fabric GitHub page.
